docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -179,14 +179,69 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Composants" href="/fr/guides/components/" />
 </CardGrid>
 
+### Asides
+
+Les encarts (également connues sous le nom de "aside" ou encore "élément en marge") sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
+
+Un `<Aside>` peut avoir un `type` optionnel de `note` (par défaut), `tip`, `caution` ou `danger`. La définition d'un attribut `title` remplace le titre par défaut de l'encart.
+
+````mdx
+# src/content/docs/example.mdx
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Un encart par défaut sans titre personnalisé.</Aside>
+
+<Aside type="caution" title="Attention !">
+	Un avertissement est mis de côté *avec* un titre personnalisé.
+</Aside>
+
+<Aside type="tip">
+
+D'autres contenus sont également pris en charge dans les encarts.
+
+```js
+// Un extrait de code, par exemple.
+```
+
+</Aside>
+
+<Aside type="danger">Ne donnez votre mot de passe à personne.</Aside>
+````
+
+Le code ci-dessus génère ce qui suit sur la page :
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Un encart par défaut sans titre personnalisé.</Aside>
+
+<Aside type="caution" title="Watch out!">
+  Un avertissement est mis en encart *avec* un titre personnalisé.
+</Aside>
+
+<Aside type="tip">
+
+  D'autres contenus sont également pris en charge dans les encarts.
+
+  ```js
+  // Un extrait de code, par exemple.
+  ```
+
+</Aside>
+
+<Aside type="danger">Ne donnez votre mot de passe à personne.</Aside>
+
+Starlight fournit également une syntaxe personnalisée pour rendre les encarts dans Markdown et MDX comme alternative au composant `<Aside>`.
```

Comment:
```suggestion
Starlight fournit également une syntaxe personnalisée pour afficher des encarts dans du contenu Markdown et MDX comme alternative au composant `<Aside>`.
```

We usually try to avoid "rendre" in French to translate "render".